lib/server-impl.js

```diff
@@ -35,12 +36,29 @@ function createApp(options) {
         stores.clientMetricsStore
     );
 
+    const impex = (config.importExport = new ImportExport(config));
+    if (config.importFile) {
+        await impex.importFile({
+            importFile: config.importFile,
+            dropBeforeImport: config.dropBeforeImport,
+            userName: 'importer',
+        });
+    }
+
     const server = app.listen({ port: options.port, host: options.host }, () =>
         logger.info(`Unleash started on port ${server.address().port}`)
     );
 
-    return new Promise((resolve, reject) => {
-        server.on('listening', () => resolve({ app, server, eventBus }));
+    return await new Promise((resolve, reject) => {
+        server.on('listening', () =>
+            resolve({
+                app,
+                server,
+                eventBus,
+                import: impex.import.bind(impex),
```

Comment:
Could we just return the `impex?` instance?

Reply:
I'd like a better name than impex though. Maybe just `importExport`